.gitignore

```diff
@@ -1 +1,2 @@
-.vscode/
\ No newline at end of file
+.vscode/
+*.pem
```

Comment:
ahh yes very important, thank you!

Since we're adding to gitignore, a couple others that might be helpful:

```
.DS_Store
.idea/
```

Reply:
good ideas!